pkg/agent/qrm-plugins/cpu/dynamicpolicy/policy.go

```diff
@@ -199,6 +201,10 @@ func NewDynamicPolicy(agentCtx *agent.GenericContext, conf *config.Configuration
 		transitionPeriod:              30 * time.Second,
 	}
 
+	if conf.EnableMBM {
+		policyImplement.mbmController = NewMBMController(conf.MBMThresholdPercentage, conf.MBMScanInterval)
```

Comment:
we shoud paas policyImplement.emitter and policyImplement.metaServer into mbmController in order to 
1. emit metrics during running
2. rerfer external mgr to submit actions

Reply:
thanks for the advice. Will add them as suggested.